pkg/controller/pod_test.go

```diff
@@ -51,9 +50,7 @@ var _ = Describe("Dynamic Attachment controller", func() {
 				multusConfigPath      = "00-multus.conf"
 			)
 
-			var err error
-			cniConfigDir, err = ioutil.TempDir("", "multus-config")
-			Expect(err).ToNot(HaveOccurred())
+			cniConfigDir = path.Join(os.TempDir(), "multus-config")
```

Comment:
A kind of pedantic. Although these changes are not equivalent (the original would create a random dir e.g. ```/tmp/multus-config<xyz>``` vs simply ```/tmp/multus-config``` with the changes), I think it should be fine since this directory  would be removed anyway when go```AfterEach(func() ```will get called. A equivalent change would be to obviously to replace go```ioutil.TempDir``` with go```os.MkdirTemp``` for go > 1.17

Reply:
Done.